strawberry/schema/schema_converter.py

```diff
@@ -411,6 +411,19 @@ def is_type_of(obj: Any, _info: GraphQLResolveInfo) -> bool:
     def from_resolver(
         self, field: StrawberryField
     ) -> Callable:  # TODO: Take StrawberryResolver
+        field.default_resolver = self.config.default_resolver  # type: ignore
+
+        if field.is_simple_field:
```

Comment:
what we did `if not field.has_resolver`?

otherwise we go with basic :)

Reply:
It's not just having a resolver that determines this, also need to check if there are any permission classes (and potentially extensions in the future). 

I'll change it to basic.

Reply:
yes, my thought was that if you add a permission class the field would get a resolver automatically, but that's probably confusing ☺️